web/modules/custom/dpl_dashboard/src/Form/DashboardSettingsForm.php

```diff
@@ -108,6 +108,12 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#default_value' => $config->get('reservations_url') ?? '',
     ];
 
+    $form['settings']['fees_url'] = [
+      '#type' => 'url',
+      '#title' => $this->t('Fees url', [], ['context' => 'Dashboard (settings)']),
+      '#default_value' => $config->get('fees_url') ?? '',
+    ];
```

Comment:
Still find it odd with no validation. Will the app work if this is just partially filled?

Reply:
We will discuss this with fddf during the e2e-test